middle.mkd

```diff
@@ -35,12 +35,14 @@ to add or delete multiple records with one resource record (RR). MIXFR is
 DNSSEC aware thus if there is a change to RRset it knows to delete the
 covering RRISG(s), this saves the transmission of old RRsigs. 
 
-## Implicit RRSIG deletion
+## Implicit DNSSEC deletions
```

Comment:
I would rather keep Implicit RRSIG deletion and Implicit NSEC3 deletion in separate sections